app.js

```diff
@@ -3,69 +3,125 @@ import { makeStatsString } from './utils.js';
+    // console.log(headSelected);
 
     // increment the head change count state
-    
+    headChangeCount++;
+    // console.log(headChangeCount);
+
     // update the dom for the head
+    // headEl.src = `./assets/${headSelected}-head.png`;
+    headEl.textContent = '';
+    const imageEl = document.createElement('img');
+    imageEl.src = `./assets/${headSelected}-head.png`;
+    imageEl.style.height = '200px';
+    headEl.append(imageEl);
+
 
     // update the stats to show the new count
+    displayStats();
 });
 
 
 middleDropdown.addEventListener('change', () => {
     // get the value of the middle dropdown
-
+    const middleSelected = middleDropdown.value;
     // increment the middle change count state
-    
+    middleChangeCount++;
     // update the dom for the middle
+    // middleEl.src = `./assets/${middleSelected}-middle.png`;
+    middleEl.textContent = '';
+    const imageEl = document.createElement('img');
+    imageEl.src = `./assets/${middleSelected}-middle.png`;
+    imageEl.style.height = '200px';
+    middleEl.append(imageEl);
+
 
     // update the stats to show the new count
+    displayStats();
 });
 
 
 bottomDropdown.addEventListener('change', () => {
     // get the value of the bottom dropdown
-
+    const bottomSelected = bottomDropdown.value;
     // increment the bottom change count state
-    
+    bottomChangeCount++;
     // update the dom for the bottom
+    // bottomEl.src = `./assets/${bottomSelected}-pants.png`;
+    bottomEl.textContent = '';
+    const imageEl = document.createElement('img');
+    imageEl.src = `./assets/${bottomSelected}-pants.png`;
+    imageEl.style.height = '200px';
+    bottomEl.append(imageEl);
+
 
     // update the stats to show the new count
+    displayStats();
+
 });
 
 catchphraseButton.addEventListener('click', () => {
     // get the value of the catchphrase input
+    const userCatchphrase = catchphraseInput.value;
     
     // push the new catchphrase to the catchphrase array in state
-    // update the dom for the bottom
+    catchphraseArray.push(userCatchphrase);
+
     // clear out the form input's value so it's empty to the user
-    // update the dom to show the new catchphrases (call a function to do this work)
+    catchphraseInput.value = '';
 
+    // update the dom to show the new catchphrases (call a function to do this work)
+    displayCatchphrases();
 });
 
+//FUNCTIONS
+
 function displayStats() {
-    // change the text contentof the reportEl to tell the user how many times they've changed each piece of the state
-    const statsString = makeStatsString(); // call this function with the correct arguments
+    // change the text content of the reportEl to tell the user how many times they've changed each piece of the state
+
+    reportEl.textContent = '';
+    
+    const statsString = makeStatsString(headChangeCount, middleChangeCount, bottomChangeCount); // call this function with the correct arguments
+
+    reportEl.append(statsString);
```

Comment:
Nice job navigating all of the "s" going on here.  "S" is the source of most typo errors in programming.